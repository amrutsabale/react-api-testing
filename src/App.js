import './App.css';
import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
import Posts from './Posts';
import Todos from './Todos';
function App() {
  return (
    <div className="app">
      <Router>
        <Switch>
          <Route path="/posts"  >
            <Posts />
          </Route>
          <Route path="/todos" >
            <Todos />
          </Route>
          <Route path="/" >
            <div>
              <Link to="/todos">
                Go to Todos
            </Link>
            </div>
            <div>
              <Link to="/posts">
                Go to Posts
            </Link>
            </div>
          </Route>
        </Switch>
      </Router>
    </div>
  );
}

export default App;
