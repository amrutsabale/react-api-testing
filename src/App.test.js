import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
  it('renders App corectly', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
})
