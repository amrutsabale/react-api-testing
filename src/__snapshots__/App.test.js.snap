// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App corectly 1`] = `
<div>
  <div
    class="app"
  >
    <div>
      <a
        href="/todos"
      >
        Go to Todos
      </a>
    </div>
    <div>
      <a
        href="/posts"
      >
        Go to Posts
      </a>
    </div>
  </div>
</div>
`;
